### ISSUE TITLE
Email template variables for order list

### ISSUE THREAD

**Issue by ultrasites** [Dolibarr/dolibarr#19457]
# Instructions
One email template variable to show the order list in html emails, e.g. __ORDERLIST__

Rendered Example:
| Position   |      Product      |  Amount | Unit | Price |
|----------|:-------------:|------:|-------:|-------:|
| 1 |  ProductA | 5 | piece| 35,00$|
| 2 |    ProductB   |   1 | piece |10,50$|
| 3 | ProductB |    32 | piece | 332,45$|

# Feature Request
[*Short description*]

## Use case
[*Verbose description*]

## Suggested implementation
[*Verbose description*]

## Suggested steps
[*List of tasks to achieve goal*]

**Comment by ultrasites** [comment]
it is still relevant - feature request

**Comment by ultrasites** [comment]
Hi, any news?

**Comment by ultrasites** [comment]
@eldy here is a PR: https://github.com/Dolibarr/dolibarr/pull/25683

**Comment by atm-maxime** [comment]
Hi. On which document would you send an email with an order list ?

**Comment by ultrasites** [comment]
@atm-maxime order/on order confirmation

**Comment by atm-maxime** [comment]
Ho ok sorry didn't get it right. I thought it was an order list but it's the order line list. Maybe your variable name confused me.

**Comment by ultrasites** [comment]
Ah ok, no it should be a detail list of an order

**Comment by AshleyButcher** [comment]
Would it not be better to have the possibility to do XSLT email templates for this functionality? It would provide more flexibility when dealing with more complex email template layouts.

**Comment by ultrasites** [comment]
I have not seen a 

> Would it not be better to have the possibility to do XSLT email templates for this functionality? It would provide more flexibility when dealing with more complex email template layouts.

I have not seen any XSLT Template in this project. Only variables exists for ODT Templates or string replacements for emails. Are you confirm?

**Comment by AshleyButcher** [comment]
@ultrasites You're right - there isn't (yet) XSLT support for templates in Dolibarr, but it would be logical to head this way for flexibility. Large ERP systems allow for XSLT email template support for exactly this purpose (to provide richer data, especially tabular data). I was hoping to open a discussion in this direction.
